core-plugins/src/main/java/io/cdap/plugin/transform/ProjectionTransform.java

```diff
@@ -101,6 +102,11 @@ public ProjectionTransform(ProjectionTransformConfig projectionTransformConfig)
     this.projectionTransformConfig = projectionTransformConfig;
   }
 
+  private static final String DROP = "drop";
```

Comment:
move these into the ProjectionTransformConfig

Reply:
Fixed